website/static/js/filepage/index.js

```diff
@@ -157,6 +157,71 @@ var FileViewPage = {
         }, 1000);
 
         self.mfrIframeParent = $('#mfrIframeParent');
+
+        $.fn.editable.defaults.mode = 'inline';
+
+        if(self.canEdit) {
+            var $fileName = $('#fileName');
+            var conflict = 'warn';
+            var to = {
+                data: {
+                    nodeId: self.node.id,
+                    path: null,
+                    provider: self.file.provider,
+                }
+            };
+            var from = JSON.parse(JSON.stringify(to));
```

Comment:
What is the purpose of this line? It seems to be applying two equal and opposite conversions.